app/src/main/java/com/example/teamproject/MapFragment.java

```diff
@@ -23,6 +25,16 @@
     MapView mMapView;
     View mView;
 
+    LocationResult locationResult = new LocationResult(){
+        @Override
+        public void gotLocation(Location location){
+            //Got the location!
+        }
+    };
+    MyLocation myLocation = new MyLocation();
```

Comment:
I would store this object within the `MyLocation` class. No need to expose this to the client `MapFragment`